machine_learning/linear_regression.py

```diff
@@ -3,14 +3,16 @@
 predictive analysis. The idea is pretty simple: we have a dataset and we have
 features associated with it. Features should be chosen very cautiously
 as they determine how much our model will be able to make future predictions.
-We try to set the weight of these features, over many iterations, so that they best
-fit our dataset. In this particular code, I had used a CSGO dataset (ADR vs
-Rating). We try to best fit a line through dataset and estimate the parameters.
+We try to set the weight of these features, using "sum of rectangular area
+over sum of square area" method which is a direct method.
+In this particular code, I had used a CSGO dataset (ADR vs Rating).
+We try to best fit a line through dataset and estimate the parameters.
 """
 import numpy as np
 import requests
 
 
+# Function to collect the CSGO dataset
```

Comment:
```suggestion
```
This comment is unnecessary because we already have a docstring for the function.